src/components/Table/index.jsx

```diff
@@ -72,12 +73,12 @@ const Table = ({
 
   const tableRef = useCallback(
     table => {
-      if (fixedHeight) {
-        if (table !== null) {
-          resizeObserver.current.observe(table?.parentNode);
-        } else {
-          if (resizeObserver.current) resizeObserver.current.disconnect();
-        }
+      if (!fixedHeight) return;
+
+      if (table !== null) {
+        resizeObserver.current.observe(table?.parentNode);
+      } else {
+        if (resizeObserver.current) resizeObserver.current.disconnect();
       }
```

Comment:
Instead of repeatly accessing a nested object attribute, assign it to a variable & use it.
```suggestion
      const observer = resizeObserver.current
      if (table !== null) {
        observer.observe(table?.parentNode);
      } else {
        if (observer) observer.disconnect();
      }
```